File: atcoder/abc366a_election_2.hs
-- vicfred
-- https://atcoder.jp/contests/abc366/tasks/abc366_a
-- implementation
main :: IO ()
main = do
  [n, t, a] <- map read . words <$> getLine :: IO [Int]
  let ans = if t > a then
        if n < 2*t then "Yes" else "No"
        else if t < a then
          if n < 2*a then "Yes" else "No"
        else
          "No"
  putStrLn $ ans

File: atcoder/abc366a_election_2.cpp
// vicfred
// https://atcoder.jp/contests/abc366/tasks/abc366_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int N, T, A;
  cin >> N >> T >> A;
  if (T > N / 2 or A > N / 2) {
    cout << "Yes" << endl;
  } else {
    cout << "No" << endl;
  }
  return 0;
}
